op-node/node/config.go

```diff
@@ -64,6 +64,16 @@ type Config struct {
 
 	// [OPTIONAL] The reth DB path to read receipts from
 	RethDBPath string
+
+	// Conductor is used to determine this node is the leader sequencer.
+	ConductorEnabled    bool
+	ConductorAddr       string
+	ConductorPort       int
+	ConductorRpcTimeout time.Duration
+}
+
+func (cfg *Config) ConductorEndpoint() string {
+	return fmt.Sprintf("http://%s:%d", cfg.ConductorAddr, cfg.ConductorPort)
```

Comment:
Will this always be http?

Reply:
replaced with a full rpc endpoint string config